### ISSUE TITLE
Feature request: GiA Fullscreen Mode

### ISSUE THREAD

**Issue by Sircam19** [Ascoware/get-iplayer-automator#293]
*What steps will reproduce the problem? Be specific, and provide as much detail as possible.*
Feature request

In Mac OS would it be possible to easily invokefull screen mode or possibly even half screen mode. 

Most Mac apps now have the green "+" button to enter fullscreen mode, interestingly however, fullscreen mode can be invoked within the PVR window but NOT in the main GiA window. 

*What version of GiA are you using? On what version of OS X?*
Mac OS Catalina

*Are you using a web proxy? DNS proxy? VPN? VPS?*
N/A

*What program are you attempting to download (full title, series and episode)? **Provide only ONE program as an example.** 
N/A

*Please provide any additional information below.*
Happy to provide additional information as required / requested. Great work Mr Scott K on such an amazing and useful app that i use daily.  Very glad you were able to take over from Dinky P!!

<img width="824" alt="Screen Shot 2020-10-25 at 6 08 22 PM" src="https://user-images.githubusercontent.com/4651001/97120628-c3ec4180-16ee-11eb-9047-791e1ce2a67f.png">

**Comment by skovatch** [comment]
The reason there isn't a plus button is that this window is marked as not resizable. I never really gave it much thought; it's just always been that way.

Turning on a flag isn't enough because not all of the elements respond well to being moved around. I can do it, though. I also noticed it's easy to get a better looking window with some other settings.

**Comment by Sircam19** [comment]
Thanks Mr Kovatch for your kind attention and consideration for this issue and all of the many others. 

S

**Comment by kuriousoranj** [comment]
A few years ago, I used to hack the resources of this app to make the window resizable. But I gave up because it puts me behind when there are updates. I'm very rusty as a macOS engineer, so I never offered to help make the change to the code myself.

It would be really nice to make the window resizable. Sometimes, when I'm catching up, I have lots of episodes to download and it's a pain to have to keep scrolling up and down to see them all.
